src/features/map/MapVisualizer.tsx

```diff
@@ -1,49 +1,64 @@
 import { Box } from '@mui/material';
```

Comment:
Ogólnie do przemyslenia w tym pliku: 
- Moze te funkcje powinny byc całe podawane w propsach? (handleDoubleClick, dragEnd itd)?
- Moze warto zastosowac ten typ Either?

Reply:
Tylko jakbym całe podawał to jak miałbym dostęp do grafu pod spodem, podać jako argument lamby dodatkowy?

Either wydawał mi się tu nie odpowiedni, bo co jeśli chialisbysmy czasowo wyłączać interaktywność np na czas puszczania requesta? Wtedy istnieje stan z interactabme na false i optionami podanymi

Reply:
No albo lambde albo np funkcje wyzszego rzedu (w sensie funkcje co zwraca funckje) (preferably to 2)